sdk/openai/Azure.AI.OpenAI/src/Generated/Internal/MultipartFormDataRequestContent.cs

```diff
@@ -0,0 +1,180 @@
+﻿// <auto-generated/>
+
+#nullable enable
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using MultipartFormDataContent = System.Net.Http.MultipartFormDataContent;
```

Comment:
This line is needed until we remove the MultipartFormDataContent file in Azure.Core shared source.  We can do this once the autorest.csharp repo no longer has a dependency on this file.